### ISSUE TITLE
v248 still monitoring /proc/self/mountinfo ?

### ISSUE THREAD

**Issue by fbuihuu** [systemd/systemd#19464]
With v248:

```
# mkdir /mnt/opt
# mount -o bind /opt /mnt/opt
# systemctl status mnt-opt.mount
Unit mnt-opt.mount could not be found.
```

**Comment by poettering** [comment]
I can't reproduce the issue here. THis just works here.  Anything in the logs?

**Comment by poettering** [comment]
(in particular in debug mode?)

**Comment by fbuihuu** [comment]
Unfortunately nothing in the logs after `systemctl log-level debug; mount -o bind /opt /mnt/opt`.

**Comment by fbuihuu** [comment]
No it doesn't wake up at all...

Apparently reverting commit d586f642fd90e3bb378f7b6d3e3a64a753e51756 solved the problem.

**Comment by fbuihuu** [comment]
(sorry I currently don't have time to investigate this issue).

**Comment by poettering** [comment]
Hmm, is it possible that something on your system so frequently triggers mount table changes that the ratelimit is systematiclly hit?

**Comment by fbuihuu** [comment]
Maybe, here's the content of fstab that contains quite a few entries to mount BTRFS subvolumes:
```UUID=88b22de3-1b5d-46f3-a619-7d79b60547f4 swap swap defaults 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c / btrfs defaults 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /boot/grub2/i386-pc btrfs subvol=boot/grub2/i386-pc 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /boot/grub2/x86_64-efi btrfs subvol=boot/grub2/x86_64-efi 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /opt btrfs subvol=opt 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /srv btrfs subvol=srv 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /tmp btrfs subvol=tmp 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /usr/local btrfs subvol=usr/local 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /var/crash btrfs subvol=var/crash 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /var/lib/libvirt/images btrfs subvol=var/lib/libvirt/images 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /var/lib/mailman btrfs subvol=var/lib/mailman 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /var/lib/mariadb btrfs subvol=var/lib/mariadb 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /var/lib/named btrfs subvol=var/lib/named 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /var/lib/pgsql btrfs subvol=var/lib/pgsql 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /var/log btrfs subvol=var/log 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /var/opt btrfs subvol=var/opt 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /var/spool btrfs subvol=var/spool 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /var/tmp btrfs subvol=var/tmp 0 0
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /.snapshots btrfs subvol=.snapshots 0 0
UUID=be8e8990-6414-4b89-8f0d-c189c28dc4a0 /home xfs defaults 1 2
UUID=18c932c3-6999-4379-9aaf-e4f3a7777b5c /var/lib/machines btrfs subvol=var/lib/machines 0 0
```

But even if the ratelimit is triggered it should be removed at some point, no ?

**Comment by poettering** [comment]
I guess this will be fixed by #19811

**Comment by fbuihuu** [comment]
Indeed and applying https://github.com/systemd/systemd/pull/19811/commits/b323a7d3e6f64fb88384df921f6bfafdc5b53ea4 fix the problem.

Thanks.
